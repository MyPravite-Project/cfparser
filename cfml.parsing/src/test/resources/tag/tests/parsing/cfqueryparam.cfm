<cfquery>
	select <cfqueryparam> from table where <cfqueryparam> =<cfqueryparam>
</cfquery>