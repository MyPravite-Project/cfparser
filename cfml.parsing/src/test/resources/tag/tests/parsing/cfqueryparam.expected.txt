START:cfquery
  =========TAG=================
  <cfquery>
	select <cfqueryparam> from table where <cfqueryparam> =<cfqueryparam>
</cfquery>
  START:cfqueryparam
    =========TAG=================
    <cfqueryparam>
  END:cfqueryparam
  START:cfqueryparam
    =========TAG=================
    <cfqueryparam>
  END:cfqueryparam
  START:cfqueryparam
    =========TAG=================
    <cfqueryparam>
  END:cfqueryparam
END:cfquery
